src/firewall.py

```diff
@@ -66,7 +68,38 @@ def get_host_ip(self) -> str:
         )
         return str(ipaddress.IPv4Interface(address.strip()).ip)
 
-    def refresh_firewall(self, denylist: typing.List[FirewallEntry]):
+    def _exclude_network(
+        self,
+        networks: typing.List[NetworkT],
+        excludes: typing.List[NetworkT],
+    ) -> typing.Set[NetworkT]:
```

Comment:
Can this be the `set` type?

Reply:
Changed into list, due to

> I made them into lists, since we need to perform append operations, converting them to list will change the IPvXNetwork into IPvXAddress and fail the algo unfortunately.